e2e/env/oci.go

```diff
@@ -171,6 +172,7 @@ func (c ctx) ociEnvOption(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		testEnv := append(os.Environ(), tt.hostEnv...)
```

Comment:
(as above)

Reply:
Previously, we were setting an empty environment for the singularitycmd, without the host PATH. We need that so that squashfuse can be found.